backend/inspirehep/records/serializers/bibtex.py

```diff
@@ -16,6 +19,27 @@
 
 
 class BibtexWriter(Writer):
+    def _encode(self, text):
+        import latexcodec  # NOQA
+
+        if "$" not in text:
+            return codecs.encode(text, "ulatex+{}".format(self.encoding))
+        else:
```

Comment:
also [flat is better than nested](https://www.python.org/dev/peps/pep-0020/): you don't need the else here, as the function `return`s if it takes the `if` branch.